changelog/v1.5.0-beta21/conflicting-empty-sni-domains.yaml

```
@@ -0,0 +1,5 @@
+changelog:
+  - type: FIX
+    issueLink: https://github.com/solo-io/gloo/issues/3459
+    description: Virtual Services with overlapping SNI domains in the same gateway are invalid. This extends last week's changes to also consider cases where the sni domain is unset, and not allow multiple configs with an unset sni domain.
```

i wouldn't talk about "last week's" changes, just in case this PR or other releases get held up a couple days for unrelated reasons
```suggestion
    description: Virtual Services with overlapping SNI domains in the same gateway are invalid. This change fixes our matching rules to consider cases where the sni domain is unset, and not allow multiple configs with an unset sni domain.
```